crates/sgx_validation/src/config.rs

```diff
@@ -0,0 +1,124 @@
+// SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
+// SPDX-License-Identifier: AGPL-3.0-only
+
+use serde::de::Error;
+use serde::{Deserialize, Deserializer, Serialize};
+use sgx::parameters::Features;
+
+#[derive(Clone, Deserialize, Debug, Default, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct Config {
+    /// Values for `mrsigner` in the report body.
+    /// This is the list of public keys which have signed the Enarx binary.
+    #[serde(default)]
+    #[serde(deserialize_with = "from_hex")]
+    pub enarx_signer: Option<Vec<Vec<u8>>>,
+
+    /// Values for `features`.
+    #[serde(default)]
+    #[serde(deserialize_with = "from_features")]
+    pub features: Option<u64>,
+
+    /// Value allowed for `cpusvn`.
+    pub cpu_svn: Option<Vec<u8>>,
+
+    /// Value for `isv_svn`, do not allow versions below this.
+    pub enclave_security_version: Option<u16>,
+
+    /// Value for `isv_prodid`, do not allow versions below this.
+    pub enclave_product_id: Option<u16>,
```

Comment:
```suggestion
    /// Value for `isv_prodid`
    pub enclave_product_id: Option<u16>,
```

Reply:
I don't see a fix

Reply:
Again, the comment has to be greatly expanded or the field name should be changed.
The correlation between `isv_prodid` and `enclave_product_id` is completely opaque to the person reading the documentation of this crate

Reply:
What I wanted to say... This is product `ID` .. not product `version`. I don't know, if we want to use it as a version field.

Reply:
Fixed during our call today